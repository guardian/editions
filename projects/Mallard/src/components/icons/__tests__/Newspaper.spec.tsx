import React from 'react'
import TestRenderer, { ReactTestRendererJSON } from 'react-test-renderer'
import { Newspaper } from '../Newspaper'

describe('Newspaper', () => {
    it('should display a Newspaper icon in SVG', () => {
        const component: ReactTestRendererJSON | null = TestRenderer.create(
            <Newspaper />,
        ).toJSON()
        expect(component).toMatchSnapshot()
    })
})
