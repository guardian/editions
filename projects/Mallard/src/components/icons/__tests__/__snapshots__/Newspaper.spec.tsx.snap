// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newspaper should display a Newspaper icon in SVG 1`] = `
<RNSVGSvgView
  bbHeight="26"
  bbWidth="18"
  fill="none"
  focusable={false}
  height={26}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 26,
        "width": 18,
      },
    ]
  }
  width={18}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGGroup
      fill={
        {
          "payload": 4280098075,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        d="M1.402 24.949a.377.377 0 01-.35-.35V2.312C.455 2.453 0 3.013 0 3.679V24.6C0 25.368.63 26 1.402 26h12.58c.665 0 1.19-.456 1.366-1.051H1.402z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
      <RNSVGPath
        d="M16.259 1.051c.21 0 .35.14.35.35v20.92c0 .21-.14.35-.35.35H3.679c-.21 0-.35-.14-.35-.35V1.4c0-.21.14-.35.35-.35h12.58zm0-1.051H3.679c-.77 0-1.402.63-1.402 1.402V22.32c0 .77.631 1.401 1.402 1.401h12.58c.77 0 1.401-.63 1.401-1.401V1.4C17.66.632 17.03 0 16.26 0z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
      <RNSVGPath
        d="M13.036 14.226H5.432a.539.539 0 01-.526-.525c0-.28.246-.526.526-.526h7.604c.28 0 .525.245.525.526 0 .28-.245.525-.525.525zM14.507 11.143h-9.11a.539.539 0 01-.526-.526c0-.28.245-.525.526-.525h9.11c.28 0 .526.245.526.525 0 .316-.21.526-.526.526zM14.507 8.76h-9.11a.539.539 0 01-.526-.525v-4.31c0-.28.245-.526.526-.526h9.11c.28 0 .526.245.526.526v4.31c0 .28-.21.525-.526.525zM14.507 17.275h-9.11a.539.539 0 01-.526-.526c0-.28.245-.525.526-.525h9.11c.28 0 .526.245.526.525s-.21.526-.526.526zM11.599 20.323H5.432a.539.539 0 01-.526-.525c0-.28.246-.526.526-.526h6.202c.28 0 .526.245.526.526-.035.28-.246.525-.561.525z"
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      />
    </RNSVGGroup>
    <RNSVGDefs>
      <RNSVGClipPath
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
        name="clip0"
      >
        <RNSVGPath
          d="M0 0h17.66v26H0z"
          fill={
            {
              "payload": 4294967295,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        />
      </RNSVGClipPath>
    </RNSVGDefs>
  </RNSVGGroup>
</RNSVGSvgView>
`;
